internal/services/automation/automation_runbook_resource_test.go

```diff
@@ -144,6 +144,31 @@ func TestAccAutomationRunbook_withJobSchedule(t *testing.T) {
 	})
 }
 
+func TestAccAutomationRunbook_withJobScheduleResourceAndUpdate(t *testing.T) {
```

Comment:
Shall we move this test case to the *automation_job_schedule_resource_test.go* instead, as it is testing that the job schedule won't be impacted by the update of the run book?